src/dotnet/APIView/APIViewWeb/Helpers/PageModelHelpers.cs

```diff
@@ -471,32 +472,35 @@ public static int ComputeActiveConversationsInActiveRevision(CodeLine[] lines, R
         /// Ensure unique label for Revisions
         /// </summary>
         /// <param name="apiRevision"></param>
-        /// <param name="addType"></param>
+        /// <param name="addAPIRevisionType"></param>
+        /// <param name="addCreatedBy"></param>
+        /// <param name="addCreatedOn"></param>
+        /// <param name="addPackageVersion"></param>
         /// <returns></returns>
-        public static string ResolveRevisionLabel(APIRevisionListItemModel apiRevision, bool addType = true)
+        public static string ResolveRevisionLabel(APIRevisionListItemModel apiRevision, 
+            bool addAPIRevisionType = true, bool addCreatedBy = true, bool addCreatedOn = true, bool addPackageVersion = true)
         {
-            var label = $"{apiRevision.CreatedOn.ToString()} | {apiRevision.CreatedBy}";
+            var label = String.Empty;
+            
+            if (addCreatedBy)
+                label = $"{apiRevision.CreatedBy}";
```

Comment:
This is probably not useful for automatic review since it's always created by azure-sdk

Reply:
Added a check